src/braket/circuits/noises.py

```diff
@@ -79,9 +79,13 @@ def __init__(self, probability: float):
             ascii_symbols=["BF({:.2g})".format(probability)],
         )
 
-    def to_ir(self, target: QubitSet):
+    def to_jaqcd(self, target: QubitSet):
         return ir.BitFlip.construct(target=target[0], probability=self.probability)
 
+    def to_openqasm(self, target: QubitSet, qubit_reference_format: str):
+        target_qubit = qubit_reference_format.format(int(target[0]))
```

Comment:
Instead of hardcoding the target at `0` can we use loop through the list using `fixed_qubit_count`?

Reply:
Yes, it'd be nice to avoid hard coding values. However, the generated string requires a single element instead of a list and the indexing here makes it clear what's being used.

Reply:
I think Cody's question is valid. We can do it for each qubit in the target. Would that be wrong?

Reply:
This operator should only ever operate on a single target qubit by definition.

Reply:
I don't feel strongly about it, especially since we've done this (for a long time) in "to_jaqcd" as well.